developer-hub/src/polkadot_sdk/mod.rs

```diff
@@ -0,0 +1,126 @@
+//! # Polkadot SDK
+//!
+//! The [Polkadot SDK repository](https://github.com/paritytech/polkadot-sdk) provides the main
+//! resources needed to start building on the [Polkadot network](https://polkadot.network), a
+//! multi-chain blockchain platform that enables different blockchains to interoperate and share
+//! information in a secure and scalable way.
+//!
+//! [![StackExchange](https://img.shields.io/badge/StackExchange-Polkadot%20and%20Substrate-222222?logo=stackexchange)](https://substrate.stackexchange.com/)
+//!
+//! [![awesomeDot](https://img.shields.io/badge/polkadot-awesome-e6007a?logo=polkadot)](https://github.com/Awsmdot/awesome-dot)
+//! [![awesomeDot](https://img.shields.io/badge/polkadot-wiki-e6007a?logo=polkadot)](https://wiki.polkadot.network/)
+//! [![awesomeDot](https://img.shields.io/badge/polkadot-forum-e6007a?logo=polkadot)](https://forum.polkadot.network/)
+//!
+//! [![RFCs](https://img.shields.io/badge/fellowship-RFCs-e6007a?logo=polkadot)](https://github.com/polkadot-fellows/rfcs)
+//! [![Runtime](https://img.shields.io/badge/fellowship-runtimes-e6007a?logo=polkadot)](https://github.com/polkadot-fellows/runtimes)
+//! [![Manifesto](https://img.shields.io/badge/fellowship-manifesto-e6007a?logo=polkadot)](https://github.com/polkadot-fellows/manifesto)
+//!
+//! ## Getting Started
+//!
+//! The primary way to get started with the Polkadot SDK is to start writing a Substrate-based
+//! runtime using FRAME. See:
+//!
+//! 0. [`polkadot`], to understand what is Polkadot as a development platform.
+//! 1. [`substrate`], for an overview of what Substrate is.
+//! 2. Jump right into [`frame`] to learn about how to write a FRAME-based blockchain runtime.
+//! 3. Continue with the [`developer_hub`'s "getting started"](crate#getting-started).
```

Comment:
Do we list from 0 on purpose?

Reply:
Lol I am not sure, I usually use bullet points to avoid this. I will edit this one.